internal/gcpmanager/manager.go

```diff
@@ -1,4 +1,4 @@
-// Package gcp provides a GCP implementation of the Substratus Cloud Interface (SCI).
+// Package gcpmanager provides a GCP implementation of the Substratus Cloud Interface (SCI).
```

Comment:
Please remove any changes here. I incorporated them in my PR

Reply:
Can do.